### DISCUSSION TITLE
Selfbot

### DISCUSSION THREAD

**Discussion by thatrandomperson5** [krisppurg/dimscord#83] [Q&A] [answered]
Will there ever be selfbot support or is there all ready. It is fine if there is none and never will be, I just wanted to check.

**Comment by krisppurg** [comment, marked as answer]
Nope, there is no selfbot support in dimscord. Since it is also against the ToS and is discouraged.

Regardless implementing it on dimscord would also be an issue since there are features that are for user accounts only that are undocumented and may change without even knowing so, e.g. endpoint change or payload change. So even then that would be a hassle to implement.

**Reply by thatrandomperson5** [reply]
Ok, thanks!
